src/SFA.DAS.FindApprenticeshipTraining/Application/TrainingCourses/LocationHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.FindApprenticeshipTraining.Configuration;
+using SFA.DAS.FindApprenticeshipTraining.InnerApi.Requests;
+using SFA.DAS.FindApprenticeshipTraining.InnerApi.Responses;
+using SFA.DAS.FindApprenticeshipTraining.Interfaces;
+
+namespace SFA.DAS.FindApprenticeshipTraining.Application.TrainingCourses
+{
+    internal class LocationHelper
```

Comment:
would be better to be public, tested in isolation, simplifying tests of it's client.